templates/etc/systemd/system/node-upload.service

```diff
@@ -10,6 +10,10 @@ KillMode=process
 Restart=always
 User=www-data
 Group=www-data
+Environment="SENTRY_ENVIRONMENT=${PERM_ENV}"
+Environment="PORT=3000"
+Environment="SENTRY_DSN=${UPLOAD_SERVICE_SENTRY_DSN}"
+Environment="AWS_REGION=${AWS_REGION}"
```

Comment:
Hmm, I'm a bit confused about why only the region is needed.

Reply:
Currently, region isn't being picked up by the aws sdk from the current configuration -- the signed requests are all using the default (us-east-1).  This fixed that, though after taking a closer look we need to set `AWS_SDK_LOAD_CONFIG` to true in order for the SDK to load region from `~/aws/config`.

I'll swap that out, since I imagine we would like to couple region across all deployed services rather than have them individually configured.

Reply:
Update: setting `AWS_SDK_LOAD_CONFIG=true` seems to work; I'll do that for the reasons listed above / it means we aren't configuring regions for our system in multiple places (if we ever wanted to configure this service with a different region, we always have the ability to do so).